lib/compress/zstd_compress.c

```diff
@@ -2274,6 +2285,39 @@ static size_t ZSTD_compressBlock_internal(ZSTD_CCtx* zc,
     return cSize;
 }
 
+static size_t ZSTD_compressSuperBlock(ZSTD_CCtx* zc,
```

Comment:
@terrelln @Cyan4973  naming wise, I am thinking of changing this function to be ZSTD_compress_targetCBlockSize() and changing ZSTD_compressSuperBlock_internal() to be ZSTD_compressSuperBlock(). Does that sound okay?

Reply:
Yes, I agree, it feels better.

In term of API design, it's important to evaluate "how does the API look like in the `*.h`" ?
The naming should make sense, the scope should make sense, etc.

And I believe the new proposed naming fits better in `zstd_compress_superblock.h`.